config/samples/gcp/gcp_openshift_route.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS
+metadata:
+  name: sample-gcp
+spec:
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: test-gcp1.qe.gcp.devcluster.openshift.com
+  provider:
+    type: GCP
+  source:
+    # Incase you have multiple ingress controllers you must specify ingress controller name in the routerName
+    # so that the external dns will use the router canonical name correrponding to it to create a dns record. 
+    openshiftRouteOptions:  
+      routerName: default
+    #Source Type is route resource of OpenShift    
+    type: OpenShiftRoute
+    fqdnTemplate: 
+    - "{{.Name}}.apps.test-gcp1.qe.gcp.devcluster.openshift.com"
+  zones:
+  - test-gcp1-q6m5v-private-zone
```

Comment:
Sorry for being a bit too pedantical but GCP managed zone ID looks more like this `3651032588905568971`.

Reply:
You can put the name which is in the output of ->
[miheer@localhost Downloads]$ gcloud dns managed-zones list | grep mykrbid.qe.gcp
mykrbid-xdl4p-private-zone            mykrbid.qe.gcp.devcluster.openshift.com.            Created By OpenShift Installer  private
[miheer@localhost Downloads]$

Reply:
Are you sure it'll work with the name of the managed zone? This value will be passed to `--zone-id-filter` flag.

Reply:
Yes from my testing

Reply:
That's a bit confusing for the flag of this name but it's true: the filtering can be done using the name too: [link](https://github.com/kubernetes-sigs/external-dns/blob/master/provider/google/google.go#L180).